update_twiki.py

```diff
@@ -0,0 +1,51 @@
+import sys
+import sqlite3
+import logging
+sys.path.append('/afs/cern.ch/cms/PPD/PdmV/tools/McM/')
+from rest import McM
+
+# McM instance
+mcm = McM(dev=False, debug=False, cookie='cookie.txt')
+
+# Logger
+logging.basicConfig(format='[%(asctime)s][%(levelname)s] %(message)s', level=logging.INFO)
+logger = logging.getLogger()
+
+def main():
+    conn = sqlite3.connect('twiki.db')
+    c = conn.cursor()
+    # Create table if it does not exist
+    c.execute('''CREATE TABLE IF NOT EXISTS twiki_samples
+                 (prepid text PRIMARY KEY NOT NULL,
+                  dataset text NOT NULL,
+                  total_events integer NOT NULL,
+                  campaign text NOT NULL,
+                  resp_group text NOT NULL)''')
+
+    # Clear the table
+    c.execute('DELETE FROM `twiki_samples`')
+    conn.commit()
+
+    # Get all needed requests
+    twiki_samples_fall_18_candidates = mcm.get('requests', query='member_of_campaign=RunIIFall18pLHE')
```

Comment:
You probably want to allow different campaigns to be used here. Remove RunIIFall18pLHE and add either a list or command line argument. Best option - add a command line argument that has comma separated campaign names